src/xcode/ENA/ENA/Source/Scenes/AntigenTestProfile/Cells/QRCodeCellViewModel.swift

```diff
@@ -40,42 +39,64 @@ struct QRCodeCellViewModel {
 		return QRCodeImage
 	}
 
-	// create vCard data, CNContactVCardSerialization creates a version 3.0 output
-	var vCardData: Data {
-		let contact = CNMutableContact()
-		contact.contactType = .person
-		contact.givenName = antigenTestProfile.firstName ?? ""
-		contact.familyName = antigenTestProfile.lastName ?? ""
+	var vCardV4: String {
```

Comment:
Is there a unit test for the creation of this string?

Reply:
I added one